src/main/java/com/conveyal/r5/util/ShapefileReader.java

```diff
@@ -89,6 +93,15 @@ public ReferencedEnvelope getBounds () throws IOException {
         return source.getBounds();
     }
 
+    public List<String> numericAttributes () {
+        return features.getSchema()
+                .getAttributeDescriptors()
+                .stream()
+                .filter(d -> Number.class.isInstance(d.getType().getBinding()))
```

Comment:
I just realized my suggestion here is incorrect because the parameter is not a value of the attribute, but the Class that will hold that value. It's testing whether a Class instance is a Number which is always false. This should instead be `Number.class.isAssignableFrom(...)`.